VERSION

```diff
@@ -1 +0,0 @@
-1.3.0
```

Comment:
Shouldn't this be `1.5.0`?

Reply:
No, to my knowledge this file is deleted, since it's not required anymore